modules/apps/commerce/commerce-product-asset-categories-web/src/main/resources/META-INF/resources/js/EditAssetCategoryCPDisplayLayout.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openSelectionModal} from 'frontend-js-web';
+
+export default function ({
+	assetCategory,
+	categorySelectorUrl,
+	itemSelectorUrl,
+	locale,
+	portletNamespace,
+}) {
+	const displayPageItemRemove = window.document.querySelector(
+		`#${portletNamespace}displayPageItemRemove`
+	);
+
+	const displayPageNameInput = window.document.querySelector(
+		`#${portletNamespace}displayPageNameInput`
+	);
+
+	const pagesContainerInput = window.document.querySelector(
+		`#${portletNamespace}pagesContainerInput`
+	);
+
+	window.document
+		.querySelector(`#${portletNamespace}chooseDisplayPage`)
+		.addEventListener('click', (_event) => {
```

Comment:
We should (usually) not chain invocations and null-check after dom query.

Also, `_event` looks unused.